emoji/src/main/res/values/library_emoji_strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="UnusedResources">
```

Comment:
I think it's a lint bug.

`emoji/lint.xml` contains this:

```xml
<issue id="UnusedResources">
    <ignore regexp="library_emoji_strings.xml"/>
</issue>
```

https://issuetracker.google.com/issues/37981147